content/10.diagnostics.md

```diff
@@ -100,9 +117,7 @@ Specifically, the individuals at risk include:
 - people with severe symptoms
 - people showing mild symptoms that have been in contact with a person who has developed the infection
 - people with underlying health conditions
-- pregnant women
 
-However, this method of testing administration does not detect a high proportion of infections.
-This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home.
-Furthermore, individuals for whom the origin of the infection is uncertain are not tested.
+However, this method of testing administration does not detect a high proportion of infections, and does not allow for test-and-trace methods to be used.
```

Comment:
```suggestion
However, this method of testing administration does not detect a high proportion of infections and does not allow for test-and-trace methods to be used.
```
Is test-and-trace currently defined anywhere in the text? If you don't mind checking and including an explanation if it isn't defined yet, that would be really helpful!

Reply:
I will add that elsewhere in the next revision.